tests/acceptance/helpers/webdavHelper.js

```diff
@@ -210,7 +209,8 @@ exports.getSkeletonFile = function (filename) {
       const headers = httpHelper.createAuthHeader('admin')
       const element = join(dir.trim(), filename)
       return fetch(
-        client.globals.backend_url + `/ocs/v2.php/apps/testing/api/v1/file?file=${encodeURIComponent(element)}&absolute=true&format=json`,
+        join(client.globals.backend_url,
+          `/ocs/v2.php/apps/testing/api/v1/file?file=${encodeURIComponent(element)}&absolute=true&format=json`),
```

Comment:
```suggestion
          '/ocs/v2.php/apps/testing/api/v1/', `file=${encodeURIComponent(element)}&absolute=true&format=json`),
```

Or, even better, separate the query params in a separate variable.

Reply:
fixed